### PR TITLE
Add "deprovision" management command to clear out user data from the DB

### PR DESCRIPTION
**Pull request by jamalex**
### Summary

Needed for when an implementing or hardware partner (Nalanda, World Possible, etc) provisions a device or device image with content, but then wants to clear out the temporary facility and user data and allow the end user to provision the device from scratch (but without needing to re-import content).

### Reviewer guidance

To test, make sure to back up your database (if it has anything you care about). Then, run `kolibri manage deprovision` and answer the prompts. Then start the server back up and check that you're taken to the onboarding wizard. After you complete the wizard, the content page should still show whatever content you had in there before.

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [x] If there are any front-end changes, before/after screenshots are included
- [x] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [x] Automated test coverage is satisfactory
- [x] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/auth/management/commands/deprovision.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        # safest not to run this command while the server is running
+        status_code, _ = server.get_urls()
+        if status_code == server.STATUS_RUNNING:
+            logging.error("The Kolibri server is currently running. Please stop it and then re-run this command.")
+            sys.exit(1)
+
+        # ensure the user REALLY wants to do this!
+        confirm_or_exit("Are you sure you wish to deprovision your database? This will DELETE ALL USER DATA!")
+        confirm_or_exit("ARE YOU SURE? If you do this, there is no way to recover the user data on this device.")
```
Comment (ralphiee22) [inline kolibri/auth/management/commands/deprovision.py:69]: maybe add one more so the user can be really sure? (kidding)

**Review by ralphiee22 — APPROVED** [review]
Manually tested.

**Comment by ralphiee22** [comment]
is this something we want to include in the documentation? or is it more we just pass this info along to our partners?

**Review by indirectlylit — APPROVED** [review]
server running check doesn't work with devserver. I filed a follow-up issue here: #3902

> is this something we want to include in the documentation?

I think we probably do want to include it, right @jamalex ? (cc @radinamatic)

**Comment by radinamatic** [comment]
If I understand correctly, this command is needed only for higher-level implementing/hardware partners who make/clone multiple devices for their deployments. As such, I'm not sure if the correct place for documenting this command is the general end-user documentation.

We should rather start another document for this type of partners and collect all the special-needs commands developed for them.

**Comment by indirectlylit** [comment]
sounds fine to me!

**Comment by fisherbryan** [comment]
Hi @radinamatic 
A technical document would be perfect that would assist us in remote support to collect data from the command prompt. Please consider this greatly! We are using it as the main part of business and to clone  multiple local servers, but we are unable to have a sufficient way of changing facility and customising a centre using it. 

We have just over 50 centre with the same Facility name. We would like to be able to rename a facility with out having to re-download the same content . And also move users + their data over to a new device should the current hardware device fail for any reason. Some technical documentation will chnge our lives :)

**Pull request merged**